helprequest/views.py

```diff
@@ -47,9 +47,22 @@ class RequestHelp(FormView):
     """
     TODO: on save of the help request, link the user requesting help to the patient fild of the help request
     """
+
+
     template_name = 'help_request.html'
     form_class = HelpRequestForm
     success_url = 'status'
+    def gethelp(request):
```

Comment:
Where is this function being called/used?

Reply:
yeah just deleted it...it's not being used elsewhere